### ISSUE TITLE
Assistant did not work after "17th nov 2020" update

### ISSUE THREAD

**Issue by DJPatron** [exex4/xx4-bfv-aimassist#4]
I think offsets was changed. Should we patiently wait for the fix, gentlemen?

**Comment by mon5termatt** [comment]
Lets wait 👍

**Comment by mon5termatt** [comment]
however here is the log for an easier fix 

```Traceback (most recent call last):
  File "assist.py", line 63, in <module>
    aimer.start()
  File "D:\mcbik\Desktop\xx4-bfv-aimassist-master\lib\aimer.py", line 64, in start
    BFV.process(phandle, cnt)
  File "D:\mcbik\Desktop\xx4-bfv-aimassist-master\lib\BFV.py", line 148, in process
    pm = PointerManager(pHandle)
  File "D:\mcbik\Desktop\xx4-bfv-aimassist-master\lib\PointerManager.py", line 30, in __init__
    offsets.OBFUS_MGR = self.GetObfuscationMgr()
  File "D:\mcbik\Desktop\xx4-bfv-aimassist-master\lib\PointerManager.py", line 61, in GetObfuscationMgr
    buf = ss.read()
  File "D:\mcbik\Desktop\xx4-bfv-aimassist-master\lib\MemAccess.py", line 684, in read
    cbuff = (c_char * len(self.buffer)).from_buffer(self.buffer)
TypeError: underlying buffer is not writable
```

**Comment by exex4** [comment]
Fixed in f9c3e2abdf1f9fda505fc97777660b41357d7e50
